File: Проверки/godzilla_vs_kong.py

budget = float(input())
the_extras = int(input())
clothing = float(input())

decor_price = budget * 0.10
clothing_price = the_extras * clothing 

if the_extras > 150:
    clothing_price_discounted = clothing_price * 0.10
    clothing_price -= clothing_price_discounted

tax = decor_price + clothing_price
total = abs(budget - tax)

if budget >= tax:
    print("Action!")
    print(f"Wingard starts filming with {total:.2f} leva left.")
else:
    print("Not enough money!")
    print(f"Wingard needs {total:.2f} leva more.")

File: Programming Basics with Java/Checks/GodzillaVsKong.java
package Checks;
import java.util.Scanner;

public class GodzillaVsKong 
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);

        double budget = input.nextDouble();
        int extras = input.nextInt();
        double clothing = input.nextDouble();

        double decor_price = budget * 0.1;
        double clothing_price = extras * clothing;

        if (extras > 150)
        {
            clothing_price -= clothing_price * 0.1;
        }

        double diff = budget - (decor_price + clothing_price);
        if (diff >= 0)
        {
            System.out.println("Action!");
            System.out.printf("Wingard starts filming with %.2f leva left.", diff);
        }
        else
        {
            System.out.println("Not enough money!");
            System.out.printf("Wingard needs %.2f leva more." , Math.abs(diff));
        }
    }
}
